torch/ao/quantization/quantize_pt2e.py

```diff
@@ -66,6 +69,23 @@ def prepare_qat_pt2e(
     model = _disallow_eval_train(model)
     return model
 
+_QUANT_OPS = [
+    torch.ops.quantized_decomposed.quantize_per_tensor.default,
+    torch.ops.quantized_decomposed.quantize_per_tensor.tensor,
+    torch.ops.quantized_decomposed.quantize_per_channel.default,
+]
+def _quant_node_filter(n: Node) -> bool:
+    """If there is any pure ops between get_attr and quantize op they will be const propagated
```

Comment:
This comment can be improved. It is somewhat confusing

Reply:
which part is confusing? maybe I don't need to mention the second paragraph here?